media-video/droidcam/files/droidcam-modulesloadd.conf

```diff
@@ -0,0 +1,3 @@
+snd_aloop
```

Comment:
This causes a annoying warning message at every boot if SND_ALOOP is compiled into the kernel instead of as module. Is there a way of accomplishing the same without having this load snd_aloop unconditional? In principle it should be possible to use this package without the audio support.

Reply:
Fixed.